src/Symfony/Component/Console/Command/Command.php

```diff
@@ -356,6 +356,11 @@ public function complete(CompletionInput $input, CompletionSuggestions $suggesti
         }
     }
 
+    public function getCode(): ?InvokableCommand
```

Comment:
The `InvokableCommand` class in *internal*, we have to change that in order to use it as return type hint in the public API.

Reply:
I would keep it internal though